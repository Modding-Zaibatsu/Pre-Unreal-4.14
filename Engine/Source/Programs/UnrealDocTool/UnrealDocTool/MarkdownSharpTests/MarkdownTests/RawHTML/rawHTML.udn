<span>Content</span>

This is a paragraph.
  
<div text="">
    Content
</div>

This is another paragraph.