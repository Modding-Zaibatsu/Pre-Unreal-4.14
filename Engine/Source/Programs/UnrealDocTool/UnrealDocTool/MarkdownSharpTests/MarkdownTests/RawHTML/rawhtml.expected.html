<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" dir="ltr" lang="en-US">
    <head>
    </head>
    <body>
        <p>
            <span><p>Content</p></span>
        </p>

        <p>
            This is a paragraph.
        </p>
        
        <div text="">
            <p>Content</p>
        </div>

        <p>
            This is another paragraph.
        </p>
    </body>
 </html>
